data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/KafkaSourceConfig.java

```diff
@@ -36,6 +38,23 @@ public class KafkaSourceConfig {
     @JsonProperty("authentication")
     private AuthConfig authConfig;
 
+    @JsonProperty("aws")
+    private AwsConfig awsConfig;
+
+    @JsonProperty("acknowledgments")
+    private Boolean acknowledgementsEnabled = false;
+
+    @JsonProperty("acknowledgments_timeout")
```

Comment:
While sqs does only have `acknowledgments` at the top level, I think this is a good time to make this a common config and to nest it a bit (we may need more configuration options for acknowledgments in the future)

```
acknowledgments:
  enabled: true
  timeout: "100s"
```

Reply:
There is a consideration to only have one setting which will also enable acknowledgments